chromadb/api/__init__.py

```diff
@@ -592,3 +593,542 @@ def delete_collection(
         database: str = DEFAULT_DATABASE,
     ) -> None:
         pass
+
+
+#############
```

Comment:
Should we generate this file? Should be pretty straightforward right?

Reply:
Also - For cleanliness would it make sense to split the async into seperate files?

Reply:
yep, down to split